docs/resources/network_routed.md

```diff
@@ -0,0 +1,74 @@
+---
+page_title: "cloudavenue_network_routed Resource - cloudavenue"
+subcategory: "Network"
+description: |-
+  Provides a VMware Cloud Director Org VDC routed Network. This can be used to create, modify, and delete routed VDC networks.
+---
+
+# cloudavenue_network_routed (Resource)
+
+Provides a VMware Cloud Director Org VDC routed Network. This can be used to create, modify, and delete routed VDC networks.
+
+## Example Usage
+
+```terraform
+resource "cloudavenue_network_routed" "example" {
+  name        = "OrgNetExample"
+  description = "Org Net Example"
+
+  edge_gateway_id = data.cloudavenue_edgegateway.example.id
+
```

Comment:
The datasource referenced in resource is not in documentation

Reply:
done with tf-doc-extractor 0.0.6